dashboard/src/pages/TreeDetails/Tabs/Build/BuildTab.tsx

```diff
@@ -71,12 +71,39 @@ const BuildTab = ({ treeDetailsData }: BuildTab): JSX.Element => {
   const { startIndex, endIndex, onClickGoForward, onClickGoBack } =
     usePagination(filteredContent?.length ?? 0, ITEMS_PER_PAGE);
   const intl = useIntl();
-  const cards = useMemo(
-    () => [
+  const cards = useMemo(() => {
+    const removeFilterBySection = (
+      filterSectionKey: string,
+      filterSection: TFilterKeys,
+    ): void => {
+      navigate({
+        search: previousParams => {
+          const { diffFilter } = previousParams;
+          const newFilter = structuredClone(diffFilter);
+          newFilter[filterSection] = newFilter[filterSection] ?? {};
+          const configs = newFilter[filterSection];
```

Comment:
nit: can we simply do: `const configs = newFilter[filterSelection] ?? {}`

Reply:
No because we would lose the pointer to newFilter[fitlerSection]